src/tests/strfmt.rs

```diff
@@ -181,14 +181,14 @@ fn test_ignore_missing() {
             0,
         ),
     ];
-    let f = |mut fmt: Formatter| match vars.get(fmt.key) {
```

Comment:
this does not need to be mut right? If it does, this is a breaking change.

Ditto for the other changes (except the `run_tests` helper, that can stay as FnMut).

Reply:
Reverted changes to other tests.  Added `test_mut_closure` test to test my specific use case.

Thanks.